packages/OptionButton/src/option-button.scss

```diff
@@ -0,0 +1,145 @@
+@use '~@igloo-ui/tokens/dist/base10/variables' as tokens;
+@use '~@igloo-ui/tokens/dist/fonts';
+
+:root {
+  /* Default */
+  --ids-option-button-border-color: #{tokens.$grey-400};
+  --ids-option-button-border-radius: #{tokens.$border-radius-sm};
+  --ids-option-button-font-size: #{tokens.$font-size-4};
+
+  /* Focus */
+  --ids-option-button-focus-outline-color: #{tokens.$electric-blue-600};
+
+  /* Hover */
+  --ids-option-button-hover-border-color: #{tokens.$grey-600};
+
+  /* Checked */
+  --ids-option-button-checked-box-shadow: 0 0 0.6rem #3c7dff;
+
+  /* disabled */
+  --ids-option-button-icon-container-disabled-background: #{tokens.$grey-200};
+
+  /* Icon */
+  --ids-option-button-icon-container-background: #{tokens.$electric-blue-50};
+  --ids-option-button-icon-container-height: 5.6rem;
+  --ids-option-button-icon-container-width: 6.2rem;
+
+  /* Text */
+  --ids-option-button-text-container-padding: #{tokens.$space-1} #{tokens.$space-3};
+  --ids-option-button-text-color: #{tokens.$grey-800};
+  --ids-option-button-text-font-size: #{tokens.$font-size-5};
+  --ids-option-button-text-font-weight: #{tokens.$font-weight-semi-bold};
+  --ids-option-button-text-line-height: #{tokens.$line-height-xl};
+
+  /* Description */
+  --ids-option-button-desc-color: #{tokens.$grey-600};
+  --ids-option-button-desc-font-size: #{tokens.$font-size-3};
+  --ids-option-button-desc-line-height: #{tokens.$line-height-xxl};
+}
+
+%active-icon-container {
+  background: var(--ids-option-button-icon-container-background);
+}
+
+%disabled-icon-container {
+  background: var(--ids-option-button-icon-container-disabled-background);
+}
+
+.ids-option-button {
+  display: block;
+  position: relative;
+
+  &__radio {
+    opacity: 0;
+    position: absolute;
+  }
+
+  &__label {
+    align-items: center;
+    border: 1px solid var(--ids-option-button-border-color);
+    border-radius: var(--ids-option-button-border-radius);
+    cursor: pointer;
+    display: flex;
+    font-family: var(--ids-option-button-font-family);
+    outline: none;
+    overflow: hidden;
+  }
+
+  &__radio:hover ~ &__label {
+    border-color: var(--ids-option-button-hover-border-color);
+  }
+
+  &__radio:focus ~ &__label {
```

Comment:
For the focus style use the property **shadow**and the `var(--focus)` because on safari the border radius are broken with the outline.

<img width="300" alt="Capture d’écran, le 2022-09-16 à 12 13 49" src="https://user-images.githubusercontent.com/2379339/190683583-6c34da29-55e6-4174-bcb8-6f56568fad76.png">